README.md

```diff
@@ -1,9 +1,55 @@
 # model_seg_rat_axon-myelin_bf
+---
+## Model overview
+![bf model preview img](bf_model_preview.png)
```

Comment:
We may have to ask permissions to publish this image as the dataset is not public.

Reply:
Opened an issue for this https://github.com/axondeepseg/default-BF-model/issues/3